elvis.config

```diff
@@ -3,42 +3,41 @@
    elvis,
    [
     {config,
-     [#{dirs => ["src", "test"],
+     [#{dirs => ["src"],
         filter => "*.erl",
-        rules => [{elvis_style, line_length, #{limit => 80}},
```

Comment:
Why are you going back to an older format for elvis.config? (the one with maps is the current canonical one)